### PR TITLE
Test coverage new feature

### PR DESCRIPTION
**Pull request by Dominik26-1**

### CODE REVIEW

**Comment by Dominik26-1** [comment]
Hi,
I would like to increase coverage of your tests

**Comment by Dominik26-1** [comment]
@jonbiemond Can you review it?

**Review by jonbiemond — CHANGES_REQUESTED** [review]
Hey @Dominik26-1, thanks for the PR! It looks great, just one tiny request please.

File: tests/app/test_db.py [hunk 1]
```diff
@@ -110,18 +147,16 @@ def test_config_db_url(
...
+                "test_db", "test_user", db_password, "localhost", 5432, overwrite
             )
+        if assertion:
+            mo().write.assert_called_once_with(expected_insert)
         else:
             mo().write.assert_not_called()
 
 
 class TestDBCommands:
-    """Test DB CLI commands."""
```
[the remark this answers is not in the document]
Comment (Dominik26-1) [reply tests/app/test_db.py]: of course, done

**Review by jonbiemond — APPROVED** [review]
Fantastic @Dominik26-1! Thanks for improving the test coverage!

**Pull request merged**